BUILD.md

```diff
@@ -146,33 +176,32 @@ curl https://run.linkerd.io/emojivoto.yml | bin/linkerd inject - | kubectl apply
 kubectl -n emojivoto port-forward svc/web-svc 8080:80
 
 # view details per deployment
-bin/linkerd -n emojivoto stat deployments
+bin/linkerd viz -n emojivoto stat deployments
 
 # view a live pipeline of requests
-bin/linkerd -n emojivoto tap deploy voting
+bin/linkerd viz -n emojivoto tap deploy voting
 ```
 
 #### Deploying Control Plane components with Tracing
 
 Control Plane components have the `trace-collector` flag used to enable
 [Distributed Tracing](https://opentracing.io/docs/overview/what-is-tracing/) for
 development purposes. It can be enabled globally i.e Control plane components
-and their proxies by using the `--control-plane-tracing` installation flag.
+and their proxies by using the `--set controlPlaneTracing=true` installation
+flag.
 
 This will configure all the components to send the traces at
 `linkerd-collector.{{.Namespace}}.svc.{{.ClusterDomain}}:55678`
```

Comment:
Should this be `collector.{{.Values.controlPlaneTracingNamespace}}.svc.{{.Values.clusterDomain}}:55678`?

Reply:
:+1: